tests/tainting_rules/kotlin/tainted_args.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
```

Comment:
Sorry for the confusion, `tests/tainting_rules` should not be used anymore... need to clean that up. Taint tests now go under `tests/rules/taint_*`.